website/src/app.css

```
@@ -0,0 +1,483 @@
+  --snow: #fafafa;
+  --white: #ffffff;
+  --ring-blue: rgba(59, 130, 246, 0.5);
+
+  /* Radius system: binary — 12px containers, 9999px pills */
+  --radius-container: 12px;
+  --radius-pill: 9999px;
+
+  /* Typography */
+  --font-display: 'Cabinet Grotesk', 'SF Pro Rounded', system-ui, -apple-system, sans-serif;
+  --font-body: 'Cabinet Grotesk', ui-sans-serif, system-ui, -apple-system, sans-serif;
+  --font-mono: ui-monospace, 'SFMono-Regular', Menlo, Monaco, Consolas, monospace;
```

Stylelint (17.7.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)